packages/dokeysto/dokeysto.3.0.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "dokeysto"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
The `dune-project` file in your repository says this package requires at least dune 1.11